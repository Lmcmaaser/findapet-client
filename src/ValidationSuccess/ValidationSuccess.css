.success {
  background-color: #FFFD8F;
  border: 3px solid black;
  display: block;
  font-size: 20px;
  font-weight: bold;
  height: 80px;
  padding: 6px;
  text-align: center;
  width: 300px;
}

.x-button {
  padding: 5px 10px 5px 10px;
}
