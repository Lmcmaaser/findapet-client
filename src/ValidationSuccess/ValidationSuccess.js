import React from 'react';
import './ValidationSuccess.css';


export default function ValidationSuccess(props) {
  if(props.message) {
    return (
      <div className="success">{props.message}
        <button className="x-button" onClick={props.changeMessage}>
        X
        </button>
      </div>
    );
  }
  return <></>
}
