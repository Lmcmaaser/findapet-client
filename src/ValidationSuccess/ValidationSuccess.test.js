import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ValidationSuccess from './ValidationSuccess'

it('renders a .success by default', () => {
  const wrapper = shallow(<ValidationSuccess />)
  expect(toJson(wrapper)).toMatchSnapshot()
})

it('renders without crashing', () => {
  shallow(<ValidationSuccess />);
});
