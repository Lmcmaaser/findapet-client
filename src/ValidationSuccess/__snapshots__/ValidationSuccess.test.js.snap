// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a .success by default 1`] = `<Fragment />`;
